lib/player.rb

```diff
@@ -0,0 +1,8 @@
+class Player
+  attr_accessor :name, :symbol
```

Comment:
change attr_accessor to attr_reader cause you don't need to write the name and symbol you are just reading.